lib/minitest/all/minitest.rbi

```diff
@@ -0,0 +1,93 @@
+# typed: strong
+
+module Minitest
+  class Test < Runnable
+    include Minitest::Assertions
+    include Minitest::Reportable
+  end
+end
+
+module Minitest::Assertions
+  extend T::Sig
+
+  sig { params(test: T.untyped, msg: T.nilable(String)).returns(TrueClass) }
+  def assert(test, msg = nil); end
+
+  sig do
+    params(
+      exp: Object,
```

Comment:
It's possible you want to use `BasicObject` instead of `Object` in all of these sigs, if you want to say "you can pass literally anything here" (`class Object < BasicObject` in Ruby)

Reply:
sounds right! addressed in https://github.com/sorbet/sorbet-typed/pull/91/commits/cdda2a566822e10c511db77dade8cecb3cbee910